### ISSUE TITLE
Cannot take out Apiarist's Backpack from AE2 syste

### ISSUE THREAD

**Issue by MayhemGang** [NillerMedDild/Enigmatica2Expert#1135]
**Modpack version**
1.62 and past

**Can the issue be reproduced?**
unknown
**Logs**
Input pastebin link to crash-report or latest log, if applicable

**Issue**
As title says.

If this is a feature request, feel free to delete the template.

**Comment by MayhemGang** [comment]
This also causes a crash if tried multiple times.

**Comment by NillerMedDild** [comment]
Please provide the crash log and latest log.

**Comment by MayhemGang** [comment]
It crashes but doesn't generate a log. Let me try again.

**Comment by MayhemGang** [comment]
https://pastebin.com/05GtvX7u
I wonder if its this.

**Comment by NillerMedDild** [comment]
That doesn't really help, provide the latest.log please.

**Comment by MayhemGang** [comment]
Found it, but it contains other stuff from mshutting it down, etc.
Also contains my name.
The file is also larger than 512kb.

**Comment by MayhemGang** [comment]
[latest.log](https://github.com/NillerMedDild/Enigmatica2Expert/files/3102490/latest.log)

**Comment by NillerMedDild** [comment]
Alright, there's nothing in there either, I'll see if I can reproduce it - Thanks.

**Comment by NillerMedDild** [comment]
I cannot reproduce this issue.
